changes for checking queuesize cronjob

avi-hmcts opened the pull request.

luigibk commented on Dockerfile:
```diff
@@ -1,22 +1,22 @@
+    chmod 0755 /usr/sbin/crond; \
 else \
-    apk --no-cache add exim logrotate && \
+    apk --no-cache add exim logrotate bash && \
```
why do you need bash?

luigibk commented on Dockerfile:
```diff
@@ -1,22 +1,22 @@
     chmod 777 /var/spool/exim && \
-    chmod 0755 /usr/sbin/exim; \
+    chmod 0755 /usr/sbin/exim && \
+    chmod 0755 /usr/sbin/crond; \
```
Why is crond added everywhere with no oprtion to exclude it?

luigibk commented on Dockerfile:
```diff
@@ -35,4 +35,7 @@ ENV LOCAL_DOMAINS=@ \
+ADD crontab.txt /crontab.txt
+ADD mailqueue.sh /mailqueue.sh
+
+CMD ["exim -bdf -q15m" ; "crond -l 8 -f"]
```
I don't like the idea of running 2 processes in the same container.

luigibk commented:
I am not convinced about running a new process in the same container. Why don't we try to scrape the exim log files (which are configurable too) from a sidecar container using an exporter such as mtail?

avi-hmcts commented:
> I am not convinced about running a new process in the same container. Why don't we try to scrape the exim log files (which are configurable too) from a sidecar container using an exporter such as mtail?

As per our discussion on slack addressed your comments and updated the PR accordingly.

luigibk commented on helm/exim/values.yaml:
```diff
@@ -80,3 +80,10 @@ fluentbit:
     pullPolicy: IfNotPresent
+
+mtail:
+  enabled: true
```
enabled should be false
